.github/workflows/build.yml

```diff
@@ -0,0 +1,59 @@
+name: Build
+
+on:
+    push:
+        branches: [main, next, master, 2.5, 3.4, 4.3, 5.1, 6.2]
+    pull_request:
+    release:
+        types: [created]
+    schedule:
+        -   cron: '0 4 * * *'
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            matrix:
+                php: [7.3, 7.4]
+                composer-flags:
+                include:
+                    -   php: 7.3
+                        composer-flags: --prefer-lowest
+                    -   php: 7.3
+                        composer-flags: --stability dev
+
+        env:
+            COMPOSER_ROOT_VERSION: dev-master
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: "${{ matrix.php }}"
+                    ini-values: "phar.readonly=0"
+
+            -   name: Install dependencies
+                run: composer update ${{ matrix.composer-flags }}
+
+            -   name: Run tests (phpspec)
+                run: bin/phpspec run --format=dot
+
+            -   name: Run tests (phpunit)
+                run: ./vendor/bin/phpunit -v
+
+            -   name: Run tests (behat)
+                run: ./vendor/bin/behat --format=progress
+
+            -   name: Build PHAR
```

Comment:
Is this going to upload the PHAR from every build?

Reply:
Actually, I think it's worth to build the PHAR on every PHP version to make sure it continues building on all of them, but only publish the phar on 7.3.

Reply:
Yes I misunderstood that it'd be uploading to the release. If it's to the GH actions page that's handy